src/main/java/explorableviz/transparenttext/paragraph/TextFragment.java

```diff
@@ -1,10 +1,20 @@
 package explorableviz.transparenttext.paragraph;
 
+import com.google.gson.JsonObject;
 import explorableviz.transparenttext.variable.Variables;
+import org.json.JSONObject;
 
 public abstract class TextFragment {
     private final String value;
 
+    public static TextFragment of(JSONObject element, String computedValue) {
```

Comment:
Where is this function used and why is `computedValue` needed in the `literal` case?